scripts/rclone_update_nfo_titles.py

```
@@ -84,6 +91,49 @@ def normalize_classification(classification: str) -> str:
+    return 0, year, name
+
+
+def select_year_dirs(
+    dirs: List[str],
+    *,
+    requested_years: Optional[List[str]] = None,
+    start_from: Optional[int] = None,
+) -> Tuple[List[str], Set[str]]:
+    """筛选 root 下需要处理的一级目录。
+
+    ``temp`` 永远排除。非年份目录被视为未知年份，排序在所有数字年份之后；
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)